egs/wsj/s5/conf/wsj_tensorflow_kaldi.conf

```diff
@@ -0,0 +1,80 @@
+# Copyright 2016    Vincent Renkens
```

Comment:
need licence type (OK to say just Apache 2.0.)
I'd be happier if this file was in some subdirectory of local/, as you may need more versions of this file as you tune things.  The conf/ directory is more for "global" configuration files, not experiment-specific ones.